Tests/TuistCacheTests/Cache/CacheRemoteStorageTests.swift

```diff
@@ -15,16 +16,27 @@ final class CacheRemoteStorageTests: TuistUnitTestCase {
     var config: Config!
     var fileArchiverFactory: MockFileArchiverFactory!
     var fileArchiver: MockFileArchiver!
-    var mockFileUploader: MockFileUploader!
+    var fileClient: MockFileClient!
+    var zipPath: AbsolutePath!
 
     override func setUp() {
         super.setUp()
+
         config = TuistCore.Config.test()
+        zipPath = fixturePath(path: RelativePath("uUI.xcframework.zip"))
+
         fileArchiverFactory = MockFileArchiverFactory()
         fileArchiver = MockFileArchiver()
-        fileArchiver.stubbedZipResult = fixturePath(path: RelativePath("uUI.xcframework.zip"))
+        fileArchiver.stubbedZipResult = zipPath
         fileArchiverFactory.stubbedMakeFileArchiverResult = fileArchiver
-        mockFileUploader = MockFileUploader()
+        fileClient = MockFileClient()
+        fileClient.stubbedDownloadResult = Single.just(zipPath)
+
+        if let env = Environment.shared as? MockEnvironment {
```

Comment:
There's no need to do this check, since we are subclassing `TuistUnitTestCase`, it's safe to assume that we are working with a mock environment.